airbyte-protocol/models/src/main/java/io/airbyte/protocol/models/CatalogHelpers.java

```diff
@@ -49,10 +49,18 @@ public static AirbyteStream createAirbyteStream(String streamName, Field... fiel
     return createAirbyteStream(streamName, Arrays.asList(fields));
   }
 
+  public static AirbyteStream createAirbyteStream(String streamName, String schemaName, Field... fields) {
```

Comment:
These methods are duplicated for now; I will remove them in my next PR when schema names are removed from stream names.